addons/udes_stock/data/scheduler.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <record id="stock.ir_cron_scheduler_action" model="ir.cron">
+            <field eval="False" name="active"/>
+        </record>
+
+        <record id="ir_cron_scheduler_action_orderpoints" model="ir.cron">
+            <field name="name">Check order points</field>
+            <field name="model_id" ref="stock.model_stock_warehouse_orderpoint"/>
+            <field name="state">code</field>
+            <field name="code">
+model.check_order_points(True)
+model._cr.commit()
```

Comment:
Also not sure if this is needed, this was in 11

Reply:
Probably not needed, in 11 we had more code.